packages/linter/src/generators/init/init-migration.ts

```
@@ -26,10 +26,12 @@ export function migrateConfigToMonorepoStyle(
   projects.forEach((project) => {
     const lintTarget = findLintTarget(project);
     if (lintTarget) {
-      const projectEslintPath = joinPathFragments(
-        project.root,
-        lintTarget.options.eslintConfig || findEslintFile(tree, project.root)
-      );
+      const eslintFile =
+        lintTarget.options.eslintConfig || findEslintFile(tree, project.root);
+      if (!eslintFile) {
```

Of config does not exist, we should silently continue migrating other projects